docs/api/baseplate/lib/message_queue.rst

```diff
@@ -138,7 +138,10 @@ See ``--help`` for more info.
 
 .. automodule:: baseplate.lib.message_queue
 
-.. autoclass:: MessageQueue
+.. autoclass:: PosixMessageQueue
+   :members:
+
+.. autoclass:: InMemoryMessageQueue
    :members:
```

Comment:
I'm not very familiar with these kinds of docs, not sure if this is right

Reply:
These are [Sphinx](https://www.sphinx-doc.org/en/master/) style docs AFAIK. This syntax will autogenerate docs entries for the class and it members (using the doc stings you defined in that class).